src/components/NetworkDropdown.tsx

```diff
@@ -32,7 +32,14 @@ const Item = ({
 
 const NetworkDropdown = () => {
   const [networkConfig, setNetworkConfig] = useAtom(networkConfigAtom);
-
+  const specifiedNetworkName = new URLSearchParams(window.location.search).get('network');
+  if (specifiedNetworkName !== null && specifiedNetworkName in networkConfigs) {
```

Comment:
We should also only do this if the url param is different than the currently selected networkConfig.

Reply:
this gets a bit funny if you then want to change networks, as the url param will force you back to the original one.

Reply:
I guess we always want to clear the param. We could use https://developer.mozilla.org/en-US/docs/Web/API/History/replaceState instead of reloading the page in that case but probably not worth the complexity